### PR TITLE
Update README.md for Coastal Models and NextGen Forcings Engine

### PR DESCRIPTION
**Pull request by jduckerOWP**
Final updates to work done for the NextGen Forcings Engine and the Coastal model BMIs as they have been transferred over to the Lynker Intel repository.

[Short description explaining the high-level reason for the pull request]

## Additions

-

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [ ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The T-Route routing module converts water flux data (e.g., overland flow, later
...
...the  [Semi-implicit Cross-scale Hydroscience Integrated System Model (SCHISM)](https://schism-dev.github.io/schism/master/index.html) and [D-Flow Flexible Mesh (FM)](https://www.deltares.nl/en/software-and-data/products/delft3d-fm-suite/modules/d-flow-flexible-mesh). The initial development stages for the coastal model BMIs have been completed and their respective GitHub repositories are stored in the following pathways: SCHISM - https://github.com/LynkerIntel/SCHISM_BMI  ; DFlowFM - https://github.com/LynkerIntel/DFlowFM_BMI . Once the NextGen framework evaluation is complete with the SCHISM Total Water Level (TWL) capability, the SCHISM BMI repository will be merged with the SCHISM master repository (https://github.com/schism-dev/schism) in the near future. For the DFlowFM model, it's BMI development is an entire seperate branch of the BMI code that is incompatible with the Deltares DFlowFM master branch and that will remain the case until the DFlowFM EC-module is redeveloped entirely for BMI compatibility. For now, the NOAA DFlowFM BMI development is a frozen DFlowFM version for research and development initiatives under the NextGen project in the future with the TWL capability.
```
Comment (PhilMiller) [inline README.md]:
```suggestion
To produce total water level forecasts, OWP is adapting two coastal models to work with NextGen: the  [Semi-implicit Cross-scale Hydroscience Integrated System Model (SCHISM)](https://schism-dev.github.io/schism/master/index.html) and [D-Flow Flexible Mesh (FM)](https://www.deltares.nl/en/software-and-data/products/delft3d-fm-suite/modules/d-flow-flexible-mesh). The initial development stages for the coastal model BMIs have been completed and their respective GitHub repositories are stored in the following pathways: SCHISM - https://github.com/LynkerIntel/SCHISM_BMI  ; DFlowFM - https://github.com/LynkerIntel/DFlowFM_BMI . Once the NextGen framework evaluation is complete with the SCHISM Total Water Level (TWL) capability, the SCHISM BMI repository will be merged with the SCHISM master repository (https://github.com/schism-dev/schism) in the near future. For the DFlowFM model, its BMI development is an entire seperate branch of the BMI code that is incompatible with the Deltares DFlowFM master branch and that will remain the case until the DFlowFM EC-module is redeveloped entirely for BMI compatibility. For now, the NOAA DFlowFM BMI development is a frozen DFlowFM version for research and development initiatives under the NextGen project in the future with the TWL capability. 
```

Comment (hellkite500) [inline README.md]:
```suggestion
To produce total water level forecasts, OWP is adapting two coastal models to work with NextGen: the  [Semi-implicit Cross-scale Hydroscience Integrated System Model (SCHISM)](https://schism-dev.github.io/schism/master/index.html) and [D-Flow Flexible Mesh (FM)](https://www.deltares.nl/en/software-and-data/products/delft3d-fm-suite/modules/d-flow-flexible-mesh). The initial development stages for the coastal model BMIs have been completed and their respective GitHub repositories are stored in the following pathways: SCHISM - [https://github.com/LynkerIntel/SCHISM_BMI](https://github.com/LynkerIntel/SCHISM_BMI)  ; DFlowFM - [https://github.com/LynkerIntel/DFlowFM_BMI](https://github.com/LynkerIntel/DFlowFM_BMI) . Once the NextGen framework evaluation is complete with the SCHISM Total Water Level (TWL) capability, the SCHISM BMI repository will be merged with the SCHISM master repository (https://github.com/schism-dev/schism) in the near future. For the DFlowFM model, it's BMI development is an entire seperate branch of the BMI code that is incompatible with the Deltares DFlowFM master branch and that will remain the case until the DFlowFM EC-module is redeveloped entirely for BMI compatibility. For now, the NOAA DFlowFM BMI development is a frozen DFlowFM version for research and development initiatives under the NextGen project in the future with the TWL capability. 
```

File: README.md [hunk 2]
```diff
@@ -73,14 +73,23 @@ Finally, aorc_bmi provides forcing data to several models (e.g., CFE and PET) wh
...
 - AORC: Analysis of Record for Calibration
 - GFS: Global Forecast System
 - HRRR: High-Resolution Rapid Refresh
-- CFS: Climate Forecast System 
-
-The Forcings Engine can produce either CSV or NetCDF files on a regular grid or at the catchment scale. For further information, please see the [Forcings Engine repo](https://github.com/NOAA-OWP/ngen-forcing).
+- RAP: Rapid Refresh
+- CFS: Climate Forecast System
+- NAM: North American Mesoscale Forecast System
+- WRF-ARW: Weather Research and Foecast Model- Advanced Research
```
Comment (PhilMiller) [inline README.md:84]:
```suggestion
- WRF-ARW: Weather Research and Forecast Model- Advanced Research
```

File: README.md [hunk 3]
```diff
@@ -73,14 +73,23 @@ Finally, aorc_bmi provides forcing data to several models (e.g., CFE and PET) wh
...
+- MRMS: Multi-Radar/Multi-Sensor System
+- Stage IV: Quantitative Precipitation Estimates
+- NWM: NWM Retrospective Forcing Files
+- NBM: National Blended Models
```
Comment (PhilMiller) [inline README.md:88]:
```suggestion
- NBM: National Blend of Models
```
Per https://vlab.noaa.gov/web/mdl/nbm

**Review by hellkite500 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -73,14 +73,23 @@ Finally, aorc_bmi provides forcing data to several models (e.g., CFE and PET) wh
...
+- NDFD: National Digital Forecast Database
+- ERA5: ERA5-Interim Reanalysis Dataset
+
+The NextGen Forcings Engine BMI can produce a single netcdf forcing file for a regular grid, catchment scale, or an unstructured mesh or act as a forcings provider BMI under the NextGen framework. The NextGen lumped forcings driver can soley produce NextGen eligble forcing netcdf/csv forcing files for a small range of forcing datasets (AORC, GFS, HRRR, CFS) as supplementary tool for the community using a fast regridding method called ExactExtract if desired. More tools in the Forcing Engine repository include pre-processing domain configurations (hydrofabric geopackages, coastal meshes) into ESMF-compliant netcdf files as well as a suite of Python scripts for downloading forcing datasets off the NOMADs operational server. For further information, please see the [Forcings Engine repo](https://github.com/NOAA-OWP/ngen-forcing).
```
Comment (PhilMiller) [inline README.md:92]:
```suggestion
The NextGen Forcings Engine BMI can produce a single NetCDF forcing file for a regular grid, catchment scale, or an unstructured mesh or act as a forcings provider BMI under the NextGen framework. The NextGen lumped forcings driver can also produce NextGen eligible forcing NetCDF/CSV forcing files for a small range of forcing datasets (AORC, GFS, HRRR, CFS) as supplementary tool for the community using a fast regridding method called ExactExtract if desired. More tools in the Forcing Engine repository include pre-processing domain configurations (hydrofabric geopackages, coastal meshes) into ESMF-compliant NetCDF files as well as a suite of Python scripts for downloading forcing datasets off the NOMADs operational server. For further information, please see the [Forcings Engine repo](https://github.com/NOAA-OWP/ngen-forcing).
```

File: README.md [hunk 5]
```diff
@@ -53,7 +53,7 @@ The T-Route routing module converts water flux data (e.g., overland flow, later
...
...sm-dev.github.io/schism/master/index.html) and [D-Flow Flexible Mesh (FM)](https://www.deltares.nl/en/software-and-data/products/delft3d-fm-suite/modules/d-flow-flexible-mesh). The initial development stages for the coastal model BMIs have been completed and their respective GitHub repositories are stored in the following pathways: SCHISM - [https://github.com/LynkerIntel/SCHISM_BMI](https://github.com/LynkerIntel/SCHISM_BMI)  ; DFlowFM - [https://github.com/LynkerIntel/DFlowFM_BMI](https://github.com/LynkerIntel/DFlowFM_BMI) . Once the NextGen framework evaluation is complete with the SCHISM Total Water Level (TWL) capability, the SCHISM BMI repository will be merged with the SCHISM master repository (https://github.com/schism-dev/schism) in the near future. For the DFlowFM model, it's BMI development is an entire seperate branch of the BMI code that is incompatible with the Deltares DFlowFM master branch and that will remain the case until the DFlowFM EC-module is redeveloped entirely for BMI compatibility. For now, the NOAA DFlowFM BMI development is a frozen DFlowFM version for research and development initiatives under the NextGen project in the future with the TWL capability.
```
Comment (PhilMiller) [inline README.md:56]: Typo `it's` -> `its`

**Review by PhilMiller — COMMENTED** [review]
Noticed a handful of typos in here just as it was being merged.

**Pull request merged**